src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.js

```diff
@@ -289,12 +298,13 @@ class ActivityFeed extends React.Component<Props, State> {
 
         return (
             // eslint-disable-next-line
```

Comment:
Can we remove this line? It's causing formatting issues, since prettier isn't running for it.